### PR TITLE
add mediaqueries

### PR DESCRIPTION
**Pull request by catakul**
Made the web app responsive. (It is not finale! I just want to get a code review)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,39 +1,60 @@
 import PeriodicTable from './Components/PeriodicTable.js';
 import styled from 'styled-components';
+import {Icon} from '@iconify/react';
```
Comment (stefankorder) [inline src/App.js:3]: If you do not use a variable you should delete it.

File: src/App.js [hunk 2]
```diff
@@ -1,39 +1,60 @@
...
-      <header>
+      <StyledNavbar>
+        {/* <StyledSearch>
+          <Icon
+            icon="ant-design:search-outlined"
+            color="red  "
+            width="23"
+            height="23"
+          />
+        </StyledSearch> */}
```
Comment (stefankorder) [inline src/App.js:16]: Please delete unused code.

File: src/App.js [hunk 3]
```diff
@@ -1,39 +1,60 @@
...
+
+/* <Wrapper>
+  <App />
+</Wrapper>
+
+
+const Wrapper = styled.main`
+  max-width: 600px;
+  margin: 0 auto;
+`; */
```
Comment (stefankorder) [inline src/App.js:58]: Please delete unused code.

File: src/Components/Modal.js [hunk 4]
```diff
@@ -3,127 +3,156 @@ import styled from 'styled-components';
 // import { mdiWikipedia } from '@mdi/js';
```
Comment (stefankorder) [inline src/Components/Modal.js]: Please delete unused code.

File: src/Components/Modal.js [hunk 5]
```diff
@@ -3,127 +3,156 @@ import styled from 'styled-components';
...
+                icon="ant-design:close-square-outlined"
+                width="45"
+                height="45"
               />
+            </Styled_Close_Button>
+            <Styled_Wikipedia_Icon href={content.source}>
+              <Icon icon="fa6-brands:wikipedia-w" width="40" height="40" />
             </Styled_Wikipedia_Icon>
+
             {/* images */}
```
Comment (stefankorder) [inline src/Components/Modal.js:22]: Please delete unused code.

File: src/GlobalStyles.js [hunk 6]
```diff
@@ -10,17 +10,18 @@ export default styled.createGlobalStyle`
...
 
   main {
-    padding: 32px;
-    display: flex;
+    margin-top: 65px;
+
+    /* display: flex;
     flex-direction: column;
-    align-items: center;
+    align-items: center; */
```
Comment (stefankorder) [inline src/GlobalStyles.js]: Please delete this */

File: src/Components/PeriodicTable.js [hunk 7]
```diff
@@ -27,7 +27,7 @@ const PeriodicTable = () => {
     setIsOpen(true);
     setInfos(elementToFind);
   }
-  console.log(infos);
+  // console.log(infos);
```
Comment (stefankorder) [inline src/Components/PeriodicTable.js]: Please delete this unused code.

File: src/Components/Modal.js [hunk 8]
```diff
@@ -3,127 +3,156 @@ import styled from 'styled-components';
...
-          <Icon
-            icon="ant-design:close-square-outlined"
-            width="45"
-            height="45"
-          />
-        </Styled_Close_Button>
-        <StyledModalOuter>
-          <StyledModalInner>
-            <Styled_Wikipedia_Icon>
+        <Styled_Outer_Modal>
```
Comment (stefankorder) [inline src/Components/Modal.js]: Please think about the naming of your components.

**Review by stefankorder — APPROVED** [review]
Good work! Please take a look at my comments.

File: src/App.js (hunk 1 above)
Comment (catakul) [reply src/App.js:3]: thanks, i left it there because I want to use Icons from iconify at some point.

File: src/App.js (hunk 2 above)
Comment (catakul) [reply src/App.js:16]: same here

File: src/Components/Modal.js (hunk 8 above)
Comment (catakul) [reply src/Components/Modal.js]: I will thank you

**Pull request merged**